### ISSUE TITLE
NoMethodError: undefined method `symfony_console'

### ISSUE THREAD

**Issue by martijnhartlief** [capistrano/symfony#112]
When using:

`namespace :deploy do
  task :migrate do
    on roles(:db) do
      symfony_console('doctrine:migrations:migrate', '--no-interaction')
    end
  end
end`

**Comment by Nyholm** [comment]
Interesting. I see that it is defined in https://github.com/capistrano/symfony/blob/2.0.0-alfa2/lib/capistrano/dsl/symfony.rb

I let someone with a bit more ruby knowledge answer this.

**Comment by martijnhartlief** [comment]
Yes I see the symfony.rb file exists, and Symfony defines work for the base Capistrano functions. But when I try to use it in a rake file or in the deploy.rb, symfony_console is undefined.

Unfortunately, my ruby knowledge is not sufficient to figure out why this is happening.
